pepper-apis/src/main/java/org/broadinstitute/ddp/model/pdf/PdfTemplate.java

```diff
@@ -12,18 +12,26 @@
     private long id;
     private PdfTemplateType type;
     private byte[] rawBytes;
+    private Long languageCodeId;
 
-    PdfTemplate(long id, PdfTemplateType type, byte[] rawBytes) {
+    PdfTemplate(long id, PdfTemplateType type, byte[] rawBytes, Long languageCodeId) {
```

Comment:
Can we have the subclasses default to `en` language code id, and then we can make this non-null?